### DISCUSSION TITLE
Blank nodes

### DISCUSSION THREAD

**Discussion by miekeGR** [ternaustralia/ontology_tern#87] [Modelling]
I have been studying the Feaella worked example: https://github.com/ternaustralia/bdr-faealla-worked-example/blob/master/output.ttl , today in particular looking at the use of blank nodes for sdo:Person and tern:Taxon. I would like to get some clarification of their use in general first as the concept is still a bit hazy in my mind. My thinking on their use for sdo:Person may have been because it reduces the requirement to verify whether either a person is unique or referring to the same person if the string of a name is the same as another name. Is that correct?

I was a bit confused as to the different ways of representing the same person e.g. "Harvey, M.S." Are these two examples just saying the same thing in a different way?

Line 30:
```
ex:06cbed19-a42d-4277-85d9-b07d19ba37fd a tern:Observation ;
    void:inDataset ex:61aa8fb2-a3dc-4e85-bd0c-d4476a89a2e3 ;
    rdfs:comment "specimen taxonomic information" ;
    prov:wasAssociatedWith [ a sdo:Person ;
            void:inDataset ex:61aa8fb2-a3dc-4e85-bd0c-d4476a89a2e3 ;
            sdo:name "Harvey, M.S." ] ;
    sosa:hasFeatureOfInterest ex:5d6ba354-e144-41c6-b61f-b39f182d5ef2 ;
    sosa:hasResult _:N6de93b4337f0452ba53317a760962aec ;
    sosa:hasSimpleResult _:N6de93b4337f0452ba53317a760962aec ;
    sosa:observedProperty <http://linked.data.gov.au/def/tern-cv/70646576-6dc7-4bc5-a9d8-c4c366850df0> ;
    sosa:phenomenonTime [ a tern:Instant ;
            time:inXSDDateTimeStamp "2015-01-01T00:00:00"^^xsd:dateTimeStamp ] ;
    sosa:resultTime "2015-01-01T00:00:00"^^xsd:dateTime ;
    sosa:usedProcedure <https://linked.data.gov.au/def/bdr-cv/specimen-method> .
```
and here Line 279:
```
ex:78d7d7a6-c717-42b1-ac9a-9bd614a7279c a tern:Observation ;
    void:inDataset ex:51052e23-3c29-4afd-9986-9a2eb25bc5af ;
    rdfs:comment "specimen taxonomic information" ;
    prov:wasAssociatedWith _:N4f5358f651354bfc970778b9bed46f8b ;
    sosa:hasFeatureOfInterest ex:b4698e40-e26e-4d97-a9ba-99eff98ea148 ;
    sosa:hasResult _:Na23d443ba5ea44adba545c048d34eb4e ;
    sosa:hasSimpleResult _:Na23d443ba5ea44adba545c048d34eb4e ;
    sosa:observedProperty <http://linked.data.gov.au/def/tern-cv/70646576-6dc7-4bc5-a9d8-c4c366850df0> ;
    sosa:phenomenonTime [ a tern:Instant ;
            time:inXSDDateTimeStamp "2015-01-01T00:00:00"^^xsd:dateTimeStamp ] ;
    sosa:resultTime "2015-01-01T00:00:00"^^xsd:dateTime ;
    sosa:usedProcedure <https://linked.data.gov.au/def/bdr-cv/specimen-method> .
```
Where the prov:wasAssociatedWith blank node details start in Line 1120:
```
_:N4f5358f651354bfc970778b9bed46f8b a sdo:Person ;
    void:inDataset ex:51052e23-3c29-4afd-9986-9a2eb25bc5af ;
    sdo:name "Harvey, M.S." .
```

Then there is tern:Taxon that also is referred to with blank nodes. In the whole dataset, there are more than one dwc:record of the same taxon as indicated by the dwc:taxonConceptID, but each instant of this taxon in the rdf has its own blank node and from what I understand, the graph would not connect these records as being the same taxon when using blank nodes. See below example from Line 1124:

```
_:N6de93b4337f0452ba53317a760962aec a tern:Taxon ;
    void:inDataset ex:61aa8fb2-a3dc-4e85-bd0c-d4476a89a2e3 ;
    dwc:class "Arachnida" ;
    dwc:family "Feaellidae" ;
    dwc:genus "Feaella" ;
    dwc:kingdom "Animalia" ;
    dwc:order "Pseudoscorpiones" ;
    dwc:phylum "Arthropoda" ;
    dwc:scientificName "Feaella (Tetrafeaella) tealei" ;
    dwc:scientificNameAuthorship "Harvey, Abrams, Beavis, Hillyer & Huey, 2016" ;
    dwc:species "Feaella tealei" ;
    dwc:specificEpithet "tealei" ;
    dwc:taxonConceptID <urn:lsid:biodiversity.org.au:afd.taxon:263dc3d1-6541-4f01-bd8c-31a9760b723c> ;
    dwc:taxonRank "species" .

_:N7da05c327e0446d9b63e7bd640f691ae a tern:Taxon ;
    void:inDataset ex:bf5426b5-0d60-4ad8-9333-c17c81574042 ;
    dwc:class "Arachnida" ;
    dwc:family "Feaellidae" ;
    dwc:genus "Feaella" ;
    dwc:kingdom "Animalia" ;
    dwc:order "Pseudoscorpiones" ;
    dwc:phylum "Arthropoda" ;
    dwc:scientificName "Feaella (Tetrafeaella) tealei" ;
    dwc:scientificNameAuthorship "Harvey, Abrams, Beavis, Hillyer & Huey, 2016" ;
    dwc:species "Feaella tealei" ;
    dwc:specificEpithet "tealei" ;
    dwc:taxonConceptID <urn:lsid:biodiversity.org.au:afd.taxon:263dc3d1-6541-4f01-bd8c-31a9760b723c> ;
    dwc:taxonRank "species" .
```

Thanks in advance for any assistance.

**Comment by dr-shorthair** [comment]
I think it would be best for there to be no blank nodes.

**Comment by edmondchuc** [comment]
@miekeGR I used blank nodes here because it's really just an example. 

You are right about the `tern:Taxon` instances with different blank nodes. For things like this, when done for real, I'd recommend avoiding blank nodes. 

Same goes for referencing people, it's best to avoid using blank nodes. But I am unsure how difficult this would be and how the BDR will manage a registry of people and what identifiers you would use. Would you check in some BDR registry of people and create persistent identifiers for them? Or would you hunt the people down in ORCID and refer to their ORCID URI instead? I think these are some of the questions you would have to ask from the BDR project.

**Comment by kheerand** [comment]
Interesting point about persistent IDs for people.  In principle we should use existing IDs where possible and avoid creating _yet another_ ID.  However non-academic folks generally don't have ORCIDs so that will only cover a subset of the people (I don't think many of the environmental consultants would have ORCIDs).

A few questions arise from this,
1. What are the existing persistent IDs that we can use for a person (ORCID being one of them)?
2. Do we allow any of them or a subset of them within ABIS?
3. Does anything need to change in ABIS to be able to represent this?
4. How do we hunt down the ID for a person (this is probably a BDR question more so than a ABIS question)?

**PS: @edmondchuc should this be a different thread on its own?**

**Reply by edmondchuc** [reply]
@kheerand yes, I think this warrants its own discussion thread.

The discussion should also include persistent identifiers for organisations as well. For example, here are two identifiers from ROR.

- https://ror.org/03wxseg04 (TERN)
- https://ror.org/030c92375 (DAWE)

DAWE URI in wikidata http://www.wikidata.org/entity/Q85756401. 

And GRID https://grid.ac/institutes/grid.467741.7 (though the actual ID is `grid.467741.7`, which is not a URI)
